modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -88,7 +88,10 @@ class Instrument_Manager extends \NDB_Menu_Filter
             );
             if ($exists <= 0) {
                 $db_config = $config->getSetting('database');
-                exec("php generate_tables_sql_and_testNames.php < $new_file");
+                exec(
+                    'php generate_tables_sql_and_testNames.php < '
+                    . escapeshellarg($new_file)
```

Comment:
This is going to conflict with the PR I linked earlier so I think you should rebase.

Reply:
Did a rebase with adding escape function in the DB and added the comment suggested.